litert/vendors/intel_openvino/BUILD

```diff
@@ -0,0 +1,16 @@
+load("//litert/build_common:litert_build_defs.bzl", "litert_lib", "litert_test", "litert_dynamic_lib")
+
```

Comment:
Please clean this up. Remove unused loads. Please add a license comment to this file